sync/api/projects.py

```diff
@@ -226,14 +238,34 @@ def create_project_submission(
     return Response(result=response["result"]["submission_id"])
 
 
+def clear_cluster_report_errors(cluster_report_orig: dict) -> dict:
```

Comment:
Wes wrote this one as a workaround for some issues we were having with certain Azure cluster reports where the error messages were messing up the client, preventing submissions. The end result was a `Too much data for declared Content-Length` error. I added a doc string, let me know if it needs to be formatted differently or needs more detail

Reply:
Thanks. What comes after the colon is mostly free-form so the underscore isn't necessary, but formatting isn't so important since the leading underscore makes it no public and not on the doc page: https://synccomputingcode.github.io/syncsparkpy/

The explanation is very helpful.